core/models/templates.go

```diff
@@ -39,6 +39,7 @@ type TemplateTranslation struct {
 	Namespace_      string                   `json:"namespace"`
 	Locale_         i18n.Locale              `json:"locale"`
 	ExternalLocale_ string                   `json:"external_locale"`
+	Params_         map[string][]any         `json:"params"`
```

Comment:
once goflow is updated(https://github.com/nyaruka/goflow/pull/1199) here we'll use 

https://github.com/nyaruka/goflow/pull/1199/files#diff-3c537f38c766974be2bf1ad977caabed386a9ed74a453137b01d93f72b827e65R90

`map[string][]static.TemplateParam`

Reply:
[UPDATE] will be update when https://github.com/nyaruka/goflow/pull/1206